ReleaseNotes

```diff
@@ -25,6 +25,37 @@ Included are routines for the following algorithms:
 Most routines have all four precisions:
 single (s), double (d), single-complex (c), double-complex (z).
 
+2.9.0 - Jan 23, 2025
+    * [DEPRECATED] The MAGMA-sparse package is considered to be "Legacy Support Mode"
+      only, meaning the MAGMA-sparse component is not under active development. Many
+      sparse routines have been deprecated. Users are encouraged to consider switching
+      to the Ginkgo library (https://ginkgo-project.github.io) to find newer and
+      actively-maintained implementations of deprecated functionality.
+      Deprecated routines will be removed in the next release.
+    * [DEPRECATED] The MAGMA V1 interface is deprecated and will be removed in the next
+      release. This includes all routines specified in the headers under the "include"
+      subdirectory with the suffix "_v1.h"
```

Comment:
Should we say, "This includes the "magma.h" header and all routines specified ..."?